dataplane/icmp.cpp

```diff
@@ -0,0 +1,156 @@
+#include "icmp.h"
+
+#include "metadata.h"
+
+namespace yanet::icmp
+{
+
+namespace
+{
+const uint8_t TTL = 128;
+}
```

Comment:
We don't need an anonymous namespace since we're using that only in this cpp file.
I suggest just `constexpr uint8_t TTL = 128;`